containers/datalab/run.sh

```diff
@@ -65,6 +82,7 @@ fi
 docker run -it --entrypoint=$ENTRYPOINT \
   -p 127.0.0.1:8081:8080 \
   -v "$CONTENT/datalab:/content/datalab" \
+  $PYDATALAB_MOUNT_OPT \
```

Comment:
there was [another change that attempted this](https://github.com/googledatalab/datalab/blob/master/containers/base/build.sh#L18) during the build process, I think keeping both is redundant and might lead to a misunderstanding, should we remove that one, since this one doesn't require rebuilding the image?